test/e2e/framework/volume/fixtures.go

```diff
@@ -696,3 +697,71 @@ func GetLinuxLabel() *v1.SELinuxOptions {
 	return &v1.SELinuxOptions{
 		Level: "s0:c0,c1"}
 }
+
+// CheckVolumeModeOfPath check mode of volume
+func CheckVolumeModeOfPath(f *framework.Framework, pod *v1.Pod, volMode v1.PersistentVolumeMode, path string) {
```

Comment:
any other tests using this besides storage?

Reply:
Yep, CheckVolumeModeOfPath is used by testVolumeContent in e2evolume and testVolumeContent is used by TestVolumeClient(), TestVolumeClient is used by e2e/common

https://github.com/kubernetes/kubernetes/search?q=TestVolumeClient

Hmm, maybe I can move everything related to it down and clear the dependency for everything uses TestVolumeClient. I will create a commit for that change to see if that makes the dependency look better

Reply:
Just tried... It is too much change, almost move everything in e2evolume to storageutils.
The code has reference almost in all the function in e2evolume. I think let's make the dependency change in the next PR. And focus this PR on just restructure the testsuite. The utils definitely need some effort to clean up.

Reply:
thanks, then I am ok with this PR.